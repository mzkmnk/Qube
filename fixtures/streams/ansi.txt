# ANSI は素通し（空行はスキップ）
[31mRed[0m line
Plain line
