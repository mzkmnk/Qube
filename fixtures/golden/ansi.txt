[31mRed[0m line
Plain line
